### PR TITLE
Fix Extension

### LINKED ISSUE #45: Extension Not Working
Since the extension is not [returning anything](https://github.com/camunda/camunda-bpmn-moddle/blob/v3.2.0/lib/extension.js#L51) when checking wether a property can be copied it's not working.

### PR DESCRIPTION
**Pull request by philippfromme**
* fix extension
* improve test coverage

Requires https://github.com/bpmn-io/bpmn-js/pull/1141

Closes #45

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "all": "run-s lint test",
     "lint": "eslint .",
     "dev": "npm test -- --watch",
-    "test": "mocha -r esm -r ./test/expect --reporter=spec --recursive test"
+    "test": "mocha -r esm -r ./test/expect --reporter=spec --inspect --recursive test"
```
Comment (nikku) [inline package.json]: Is this necessary?

File: test/spec/extension.js [hunk 2]
```diff
@@ -1,67 +1,357 @@
...
-    });
-
-    var canCloneProperty = extension.canCloneProperty(subProcess, retryCycle.$descriptor);
-
-    // then
-    expect(canCloneProperty).to.be.true;
-  });
-
-});
+'use strict';
```
Comment (nikku) [inline test/spec/extension.js]: Did you by accident convert this to Windows line endings?

**Comment by nikku** [comment]
With this thing being merged we should double check the existing copy and paste issues to see if we can close some.

File: package.json (hunk 1 above)
Comment (philippfromme) [reply package.json]: Ooops. That's a leftover of me debugging in the browser.

File: test/spec/extension.js (hunk 2 above)
Comment (philippfromme) [reply test/spec/extension.js]: Yes, don't ask me how that happened.

**Comment by philippfromme** [comment]
> As we're refactoring this extension, could we migrate it to ES6 on the fly, too?

Good idea.

> Am I correct in the assumption that this change is a BREAKING one? If so, please write that in the commit. And please reference the issue you're closing in the commit message, too.

It is breaking in the sense that it's now working and it previously wasn't.

**Pull request merged**